openmetadata-ui/src/main/resources/ui/package.json

```diff
@@ -74,6 +74,8 @@
     "react-codemirror2": "^7.2.1",
     "react-context-mutex": "^2.0.0",
     "react-copy-to-clipboard": "^5.0.4",
+    "react-dnd": "14.0.2",
+    "react-dnd-html5-backend": "14.0.2",
     "react-dom": "^16.14.0",
```

Comment:
why do we need to install an external library as we already have antd?

Reply:
@ShaileshParmar11 this, the library is to support drag-and-drop functionality.
Reference https://ant.design/components/table#components-table-demo-drag-sorting